osquery/tables/yara/yara.cpp

```diff
@@ -39,71 +40,221 @@ FLAG(bool,
      true,
      "Call malloc_trim() after YARA scans (linux)");
 #endif
+
 FLAG(uint32,
      yara_delay,
      50,
      "Time in ms to sleep after scan of each file (default 50) to reduce "
      "memory spikes");
 
+HIDDEN_FLAG(bool,
+            enable_yara_sigrule,
+            false,
+            "Enable yara table extension to pass sigrule with query ");
+
+HIDDEN_FLAG(bool,
```

Comment:
This is a bit confusing, I recommend `enable_yara_string` and set it to `false` by default.

The description can read `Enable returning matched YARA strings`.